src/code/z_fbdemo_circle.cpp

```diff
@@ -152,7 +158,12 @@ void TransitionCircle_Draw(void* thisx, Gfx** gfxP) {
     // These variables are a best guess based on the other transition types.
     f32 tPos = 0.0f;
     f32 rot = 0.0f;
+#ifdef N64_VERSION
```

Comment:
no need for the N64_VERSION flag.  the scaler evaluates to 1 at the standard 4:3 aspect ratio.